go-controller/pkg/ovn/policy.go

```diff
@@ -808,13 +809,17 @@ func (oc *Controller) handlePeerPodSelector(
 		podSelector,
 		cache.ResourceEventHandlerFuncs{
 			AddFunc: func(obj interface{}) {
-				oc.handlePeerPodSelectorAddUpdate(gp, obj)
+				if err := oc.handlePeerPodSelectorAddUpdate(gp, obj); err != nil {
```

Comment:
won't AddFunc be called if / when ovn-k master restarts? So, a failure to add here could still be an error...

Reply:
yes, but at least the warning will still be there. I think that trade off is better than seeing the errors in the log all the time during normal operation. What do you think?